CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# 0.3.0-alpha.2 2018-xx-xx
+* The changelog is back!
+* Compatiblity with futures API in latest nightly
+* Code examples and doc improvements for the `AsyncReadExt`, `AsyncWriteExt` and `TryFutureExt` methods
+* Bugfix for the `write_all` combinator
+* `AsyncWrite` impl for `Cursor<T: AsMut<[u8]>>`
+* `FuturesUnordered` optimization: Since the context stores a `&LocalWaker` reference, it was possible to avoid cloning the `Arc` of the waker
+* Future-RS now uses Clippy
```

Comment:
`` s/Future-RS/`futures-rs` ``

Matches how the project is typographized (pretty sure that's not a word, but hopefully you know what I mean) in the README

Reply:
We can use futures-rs, but I don't think we should highlight it.

I'll need to change that on the website as well... (no typo there, but it uses capital letters)

Reply:
Ah yeah, checking the alpha blog post that uses non-code futures-rs to refer to the project. In that case I guess `s/Future-RS/Futures-rs` would be the way to go.

Reply:
Except this one is at the start of a sentence, either or is good.